tutorials/understanding-gov/index.md

```diff
@@ -321,7 +323,7 @@ After waiting for the voting period, you can see that the proposal has passed.
 $ simd query gov proposal 1 --output json | jq .status
 ```
 
-This is because the governance proposal weights each vote by the amount of tokens staked. Alice owns staked tokens, while Bob has no staked tokens. So Bob's vote was not taken into consideration in the tally of the result.
+This is because the governance proposal weights each vote by the amount of tokens staked. Alice owns staked tokens, while Bob has no staked tokens at the end of the voting period. So Bob's vote was not taken into consideration in the tally of the result.
```

Comment:
```suggestion
This is because the governance proposal weights each vote by the amount of tokens staked. Alice owns staked tokens, while Bob had no staked tokens at the end of the voting period. So Bob's vote was not taken into consideration in the tally of the result.
```